app/script/view_model/ModalsViewModel.js

```diff
@@ -223,7 +223,7 @@ z.viewModel.ModalsViewModel = class ModalsViewModel {
     }
```

Comment:
L215-217 need to be adjusted as well as we build the string Id on the fly there.

Reply:
Fixed